lib/edit-site-page.php

```diff
@@ -145,30 +145,11 @@ function gutenberg_edit_site_init( $hook ) {
 	}
 	$settings['styles'] = gutenberg_get_editor_styles();
 
-	$template_ids      = array();
-	$template_part_ids = array();
-	foreach ( get_template_types() as $template_type ) {
```

Comment:
Right, so this loop is used to compute `$template_ids` and `$template_part_ids` for the current page, which we're then sending to the client as part of `editSiteInitialState`. However, we also have client-side logic now that fetches these things from the REST API, so the server-side passed initial state isn't required anymore. (It could arguably provide an optimization, but as we've seen, it doesn't currently because of auto-draft generation; and there are better ways to pre-load data that is otherwise fetched from the REST API.)